### ISSUE TITLE
ABC iview

### ISSUE THREAD

**Issue by Poida321** [ytdl-org/youtube-dl#20786]
## Please follow the guide below

- You will be asked some questions and requested to provide some information, please read them **carefully** and answer honestly
- Put an `x` into all the boxes [ ] relevant to your *issue* (like this: `[x]`)
- Use the *Preview* tab to see what your issue will actually look like

---

### Make sure you are using the *latest* version: run `youtube-dl --version` and ensure your version is *2019.04.24*. If it's not, read [this FAQ entry](https://github.com/ytdl-org/youtube-dl/blob/master/README.md#how-do-i-update-youtube-dl) and update. Issues with outdated version will be rejected.
- [X] I've **verified** and **I assure** that I'm running youtube-dl **2019.04.24**

### Before submitting an *issue* make sure you have:
- [X] At least skimmed through the [README](https://github.com/ytdl-org/youtube-dl/blob/master/README.md), **most notably** the [FAQ](https://github.com/ytdl-org/youtube-dl#faq) and [BUGS](https://github.com/ytdl-org/youtube-dl#bugs) sections
- [X] [Searched](https://github.com/ytdl-org/youtube-dl/search?type=Issues) the bugtracker for similar issues including closed ones
- [X] Checked that provided video/audio/playlist URLs (if any) are alive and playable in a browser

### What is the purpose of your *issue*?
- [X] Bug report (encountered problems with youtube-dl)
- [ ] Site support request (request for adding support for a new site)
- [ ] Feature request (request for a new functionality)
- [ ] Question
- [ ] Other

---

### The following sections concretize particular purposed issues, you can erase any section (the contents between triple ---) not applicable to your *issue*

---

### If the purpose of this *issue* is a *bug report*, *site support request* or you are not completely sure provide the full verbose output as follows:

Add the `-v` flag to **your command line** you run youtube-dl with (`youtube-dl -v <your command line>`), copy the **whole** output and insert it here. It should look similar to one below (replace it with **your** log inserted between triple ```):

```
C:\>C:\youtube-dl.exe --verbose https://iview.abc.net.au/show/bluey/series/0/video/CH1702Q041S00
[debug] System config: []
[debug] User config: []
[debug] Custom config: []
[debug] Command-line args: ['--verbose', 'https://iview.abc.net.au/show/bluey/series/0/video/CH1702Q041S00']
[debug] Encodings: locale cp1252, fs mbcs, out cp437, pref cp1252
[debug] youtube-dl version 2019.04.24
[debug] Python version 3.4.4 (CPython) - Windows-7-6.1.7601-SP1
[debug] exe versions: ffmpeg N-93544-g0a347ff422, ffprobe N-93544-g0a347ff422
[debug] Proxy map: {}
[debug] Using fake IP 1.149.88.243 (AU) as X-Forwarded-For.
[abc.net.au:iview] CH1702Q041S00: Downloading JSON metadata
[abc.net.au:iview] CH1702Q041S00: Downloading webpage
ERROR: Unable to download webpage: HTTP Error 404: Not Found (caused by HTTPError()); please report this issue on https://yt-dl.org/bug . Make sure you are using the latest version; type  youtube-dl -U  to update. Be sure to call youtube-dl with the --verbose flag and include its complete output.
  File "C:\Users\dst\AppData\Roaming\Build archive\youtube-dl\ytdl-org\tmp1l3rbf78\build\youtube_dl\extractor\common.py", line 626, in _request_webpage
  File "C:\Users\dst\AppData\Roaming\Build archive\youtube-dl\ytdl-org\tmp1l3rbf78\build\youtube_dl\YoutubeDL.py", line 2227, in urlopen
  File "C:\Python\Python34\lib\urllib\request.py", line 470, in open
  File "C:\Python\Python34\lib\urllib\request.py", line 580, in http_response
  File "C:\Python\Python34\lib\urllib\request.py", line 508, in error
  File "C:\Python\Python34\lib\urllib\request.py", line 442, in _call_chain
  File "C:\Python\Python34\lib\urllib\request.py", line 588, in http_error_default
```

---

### If the purpose of this *issue* is a *site support request* please provide all kinds of example URLs support for which should be included (replace following example URLs by **yours**):
- Single video: https://www.youtube.com/watch?v=BaW_jenozKc
- Single video: https://youtu.be/BaW_jenozKc
- Playlist: https://www.youtube.com/playlist?list=PL4lCao7KL_QFVb7Iudeipvc2BCavECqzc

Note that **youtube-dl does not support sites dedicated to [copyright infringement](https://github.com/ytdl-org/youtube-dl#can-you-add-support-for-this-anime-video-site-or-site-which-shows-current-movies-for-free)**. In order for site support request to be accepted all provided example URLs should not violate any copyrights.

---

### Description of your *issue*, suggested solution and other information

ABC iView was working earlier in 2019, but has stopped working in last 8 weeks or so.

**Comment by remitamine** [comment]
```
youtube-dl https://iview.abc.net.au/show/bluey/series/0/video/CH1702Q041S00
[abc.net.au:iview] CH1702Q041S00: Downloading JSON metadata
[abc.net.au:iview] CH1702Q041S00: Downloading webpage
[abc.net.au:iview] CH1702Q041S00: Downloading m3u8 information
[download] Destination: Zoo-CH1702Q041S00.mp4
ffmpeg version n4.1.3 Copyright (c) 2000-2019 the FFmpeg developers
  built with gcc 8.2.1 (GCC) 20181127
  configuration: --prefix=/usr --disable-debug --disable-static --disable-stripping --enable-fontconfig --enable-gmp --enable-gnutls --enable-gpl --enable-ladspa --enable-libaom --enable-libass --enable-libbluray --enable-libdrm --enable-libfreetype --enable-libfribidi --enable-libgsm --enable-libiec61883 --enable-libjack --enable-libmodplug --enable-libmp3lame --enable-libopencore_amrnb --enable-libopencore_amrwb --enable-libopenjpeg --enable-libopus --enable-libpulse --enable-libsoxr --enable-libspeex --enable-libssh --enable-libtheora --enable-libv4l2 --enable-libvidstab --enable-libvorbis --enable-libvpx --enable-libwebp --enable-libx264 --enable-libx265 --enable-libxcb --enable-libxml2 --enable-libxvid --enable-nvdec --enable-nvenc --enable-omx --enable-shared --enable-version3
  libavutil      56. 22.100 / 56. 22.100
  libavcodec     58. 35.100 / 58. 35.100
  libavformat    58. 20.100 / 58. 20.100
  libavdevice    58.  5.100 / 58.  5.100
  libavfilter     7. 40.101 /  7. 40.101
  libswscale      5.  3.100 /  5.  3.100
  libswresample   3.  3.100 /  3.  3.100
  libpostproc    55.  3.100 / 55.  3.100
[hls,applehttp @ 0x558e992090c0] Opening 'http://iviewhls-i.akamaihd.net/i/playback/_definst_/_video/ch1/CH1702Q041S00MA1D1_20190327165943_,1500000,1000000,650000,500000,220000,.mp4.csmil/crypt.key?null=0&id=AgDFAUSQKef1BAc1wFz7djJQcjDz9dAVfD7Q6LKxR4y6ZZTRdT6I7mEphvnutr7FLvGAZzvUvFGwpQ%3d%3d' for reading
[hls,applehttp @ 0x558e992090c0] Opening 'crypto+http://iviewhls-i.akamaihd.net/i/playback/_definst_/_video/ch1/CH1702Q041S00MA1D1_20190327165943_,1500000,1000000,650000,500000,220000,.mp4.csmil/segment1_0_av.ts?null=0&id=AgDFAUSQKef1BAc1wFz7djJQcjDz9dAVfD7Q6LKxR4y6ZZTRdT6I7mEphvnutr7FLvGAZzvUvFGwpQ%3d%3d' for reading
Input #0, hls,applehttp, from 'http://iviewhls-i.akamaihd.net/i/playback/_definst_/_video/ch1/CH1702Q041S00MA1D1_20190327165943_,1500000,1000000,650000,500000,220000,.mp4.csmil/index_0_av.m3u8?null=0&id=AgDFAUSQKef1BAc1wFz7djJQcjDz9dAVfD7Q6LKxR4y6ZZTRdT6I7mEphvnutr7FLvGAZzvUvFGwpQ%3d%3d':
  Duration: 00:07:01.00, start: 0.100511, bitrate: 0 kb/s
  Program 0 
    Metadata:
      variant_bitrate : 0
    Stream #0:0: Video: h264 (High) ([27][0][0][0] / 0x001B), yuv420p, 1024x576 [SAR 1:1 DAR 16:9], 25 fps, 25 tbr, 90k tbn, 50 tbc
    Metadata:
      variant_bitrate : 0
    Stream #0:1: Audio: aac (LC) ([15][0][0][0] / 0x000F), 44100 Hz, stereo, fltp
    Metadata:
      variant_bitrate : 0
    Stream #0:2: Data: timed_id3 (ID3  / 0x20334449)
    Metadata:
      variant_bitrate : 0
Output #0, mp4, to 'file:Zoo-CH1702Q041S00.mp4.part':
  Metadata:
    encoder         : Lavf58.20.100
    Stream #0:0: Video: h264 (High) (avc1 / 0x31637661), yuv420p, 1024x576 [SAR 1:1 DAR 16:9], q=2-31, 25 fps, 25 tbr, 90k tbn, 90k tbc
    Metadata:
      variant_bitrate : 0
    Stream #0:1: Audio: aac (LC) (mp4a / 0x6134706D), 44100 Hz, stereo, fltp
    Metadata:
      variant_bitrate : 0
Stream mapping:
  Stream #0:0 -> #0:0 (copy)
  Stream #0:1 -> #0:1 (copy)
Press [q] to stop, [?] for help
frame=   92 fps=0.0 q=-1.0 size=     256kB time=00:00:03.71 bitrate= 564.6kbits
```

**Comment by Poida321** [comment]
Thank you and my apologies if I wasted your time. I found out the issue - my laptop clock is not keeping time (off by a day) and I think it won't allow a streaming connection due to some kind of out-of-date certificates or SSL or whatever (just a hunch, I don't really know what i'm talking about).

I updated my clock (and python version) and it's working fine now. Again my apologies and thanks for testing.
